drake/CTestCustom.cmake.in

```diff
@@ -0,0 +1,14 @@
+set(CTEST_CUSTOM_COVERAGE_EXCLUDE
+  ".*/test/.*"
+  ".*/thirdParty/.*"
+  ${CTEST_CUSTOM_COVERAGE_EXCLUDE}
+)
+
+set(CTEST_CUSTOM_MAXIMUM_NUMBER_OF_ERRORS 100)
+
+set(CTEST_CUSTOM_MAXIMUM_NUMBER_OF_WARNINGS 100)
+
+set(CTEST_CUSTOM_WARNING_MATCH
+  "(?:(?:((?:[/.]|[A-Za-z]:).+?):(-?\\d+):\\s*([Ww]arning|[Ee]rror)|<.+>:-?\\d+(?::\\s*([Ww]arning|[Ee]rror))?): (.+(?:\\n(?!\\s*(?:[Nn]otice|[Ww]arning|[Ee]rror): )[^/<\\n][^:\\n][^/\\n].+)*)|\\s*([Nn]otice|[Ww]arning|[Ee]rror): (.+))$"
+  ${CTEST_CUSTOM_WARNING_MATCH}
+)
```

Comment:
I wasn't even aware of the difference until I dug in with Bill to figure out why CDash stopped showing warnings.  The behavior is kind of messy because it evolved as use cases changed over time and quick fixes were made.  No one has ever thought through the whole design or documented all this fully.  See [this entry point](https://cmake.org/gitweb?p=cmake.git;a=blob;f=Source/CTest/cmCTestBuildHandler.cxx;hb=v3.5.2#l237) for reading the relevant code in CTest.